node/node_test.go

```diff
@@ -62,6 +62,25 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+type testBlockInfo struct {
+}
+
+func (t testBlockInfo) asApiResult() api.BlockInfo {
```

Comment:
missing pointers?

Reply:
no need, since the structure doesn't have any member

Reply:
then why do we use a struct?

Reply:
just to organize the code, to have a single instance that I'll use in all the tests